### PR TITLE
Move DLK to blueoil/converter 

### PR DESCRIPTION
**Pull request by kchygoe**
## What this patch does to fix the issue. 

Related to https://github.com/blue-oil/blueoil/issues/762 Mainly 
- dlk/python/dlk => blueoil/converter 
- dlk/tests => tests/converter 
- others along with above changes

## Link to any relevant issues or pull requests.
https://github.com/blue-oil/blueoil/issues/762

### CODE REVIEW

File: tests/converter/test_binary.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import unittest
 from os.path import join
 
-from scripts import generate_project as gp
+from blueoil.converter import generate_project as gp
```
Comment (iizukak) [inline tests/converter/test_binary.py]: If tests in `dlk/tests` if unit test, it's preffer to put these test files in `tests/unit/converter`.

**Comment by iizukak** [comment]
https://github.com/blue-oil/blueoil/blob/master/output_template/python/lmnet/nnlib.py
This file is a symlink to dlk. We should replace.

**Comment by iizukak** [comment]
`setup.py` in dlk looks just removed. It's ok?
How about `install_requires` ?

File: tests/converter/test_binary.py (hunk 1 above)
Comment (kchygoe) [reply tests/converter/test_binary.py]: oh, okey

**Comment by kchygoe** [comment]
> `PYTHONPATH`
I've forgotten it! Let me remove!
>

File: tests/converter/test_binary.py (hunk 1 above)
Comment (kchygoe) [reply tests/converter/test_binary.py]: I suppose these are not unittest... it would be better to re-define unittests for converter. can we discuss those in another issue?

**Comment by kchygoe** [comment]
Notes
- `blueoil/converter` flake8 adoption should be in another PR.
- `tests/converter` should be refactored by new unittests
- `tests/fixtures` should be removed and placed on other server as artifacts

File: tests/converter/test_binary.py (hunk 1 above)
Comment (iizukak) [reply tests/converter/test_binary.py]: @kchygoe I see. OK to put tests in `tests/converter`

File: .buildkite/dlk-pipeline.yml [hunk 2]
```diff
@@ -62,4 +54,4 @@ steps:
     env:
       BUILDKITE_CLEAN_CHECKOUT: 'true'
       DEVICE_TEST_INPUT_PATH: './output/TestCodeGenerationAarch64'
-      PYTHONPATH: 'dlk/python/dlk/scripts/pylib'
+      PYTHONPATH: '.'
```
Comment (tsawada) [inline .buildkite/dlk-pipeline.yml]: Is this necessary? I think Python might look into current directory with empty PYTHONPATH

File: blueoil/converter/frontend/base.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def read(self, path: str) -> Graph:
 
         Args:
             path (str): Path to the file to be read
-        
+
```
Comment (tsawada) [inline blueoil/converter/frontend/base.py]: Are these whitespace diffs related to the move? If not I recommend not to include them.

File: .buildkite/dlk-pipeline.yml (hunk 2 above)
Comment (kchygoe) [reply .buildkite/dlk-pipeline.yml]: No, It didn't work in the raspberry-pi when I checked. let me work on that in another issue. thx.

File: blueoil/converter/frontend/base.py (hunk 3 above)
Comment (kchygoe) [reply blueoil/converter/frontend/base.py]: Thx, but please just ignore white spaces in the diff by the review URL `&w=1`. It's too annoying to avoid that manually.

File: .buildkite/dlk-pipeline.yml (hunk 2 above)
Comment (kchygoe) [reply .buildkite/dlk-pipeline.yml]: In the dir,
```
$ DEVICE_TEST_INPUT_PATH=./output/TestCodeGenerationAarch64 python3 ./tests/device_tests/test_device.py
Traceback (most recent call last):
  File "./tests/device_tests/test_device.py", line 21, in <module>
    from blueoil.converter.nnlib import NNLib
ModuleNotFoundError: No module named 'blueoil'
```

Comment (kchygoe) [reply .buildkite/dlk-pipeline.yml]: (however, in python3 interpreter on the same dir, it works...)

File: docker/Dockerfile [hunk 4]
```diff
@@ -125,17 +125,10 @@ FROM blueoil-base AS blueoil-dev
...
-# Install dlk
-WORKDIR /home/blueoil/dlk
-RUN PYTHONPATH=python/dlk python setup.py install
-RUN chmod 777 /home/blueoil/dlk
-
 # Copy blueoil test code to docker image
 COPY tests /home/blueoil/tests
 
-ENV PYTHONPATH $PYTHONPATH:/home/blueoil:/home/blueoil/dlk/python/dlk
+ENV PYTHONPATH $PYTHONPATH:/home/blueoil
```
Comment (primenumber) [inline docker/Dockerfile]: Is it needed?

Comment (kchygoe) [reply docker/Dockerfile]: we don't use anymore! let me remove

Comment (kchygoe) [reply docker/Dockerfile]: removed

**Comment by kchygoe** [comment]
@tsawada please give me a review!

**Comment by iizukak** [comment]
@tsawada We need readability reviewer.

**Comment by kchygoe** [comment]
https://github.com/blue-oil/blueoil/issues/603
